site/docs/v1/tech/tutorials/integrate-flutter.md

```diff
@@ -0,0 +1,747 @@
+| `Name`                     | `My Flutter App`                              | Your application name, for display-only purposes.                                                                                                                            |
+| `Client Id`                | `7e3637e8-723a-42d6-9d1d-5cb36128d6f1`        | This is the Id used in every code in this article.                                                                                                                           |
+| `Client Authentication`    | `Not required`                                | Since the Flutter application is a native app, it's not considered a "confidential client" and a client secret isn't safe. It could be found by decompiling the application. |
+| `PKCE`                     | `Required`                                    | PKCE (often pronounced "pixie") stands for _"Proof of Key Code Exchange"_, which is another method of securing access and is what the Futter library will use.               |
+| `Authorized redirect URLs` | `com.fusionauth.flutterdemo://login-callback` | You’ll use this redirect URL in your Flutter application later.                                                                                                              |
+
+Click the blue floppy disk icon to save it. When properly configured, the application details screen should look like this:
+
+{% include _image.liquid src="/assets/img/blogs/securing-flutter-app/application-configuration.png" alt="Configuring the FusionAuth application." class="img-fluid" figure=false %}
+
+Now, register the desired users to the new FusionAuth application. Doing so creates a relationship between a user and the newly created application as shown in the screenshot below:
+
+{% include _image.liquid src="/assets/img/blogs/securing-flutter-app/user-registration.png" alt="Registering the user for the new application." class="img-fluid" figure=false %}
+
+If you want, you can add more users to the "Users" and register them to this application. But you can work through this tutorial with just one registered user.
+
+If you want to test with a real device, you should also set up ngrok to point to the local FusionAuth server, so that devices can access `http://localhost:9011`. While beyond the scope of this tutorial, here's [ngrok documentation](https://ngrok.com/docs#getting-started-expose) on how to do so.
+
+Next, move on to setting up the Flutter app project.
+
+## Setting up the Flutter project
+
+Setting up the Flutter project is simple and easy for every OS: just follow the instructions in the [Flutter documentation](https://flutter.dev/docs/get-started/install). But to make things easier, a quick outline of the setup steps is provided below.
+
+First, clone the Flutter SDK from GitHub by running the following command in your device terminal:
```

Comment:
Cloning the flutter sdk via git is the best way to install it?

This page seems to indicate that you can download it: https://docs.flutter.dev/get-started/install

Reply:
It seems that [macOS instructions](https://docs.flutter.dev/get-started/install/macos) don't have an easy way of installing the latest version (I've tried going through [their bucket](https://storage.googleapis.com/flutter_infra_release/) to see if they have a symlink or something but they don't). Do you think we should skip the instructions on our side at all and just say "visit these docs for instructions"?

Reply:
Yes, I think it is okay to add Flutter as a prerequisite and link to the install instructions.